R/namespace.R

```diff
@@ -71,27 +55,87 @@ roclet_clean.roclet_namespace <- function(x, base_path) {
   }
 }
 
+# NAMESPACE updates -------------------------------------------------------
+
+import_directives <- c(
+  "import",
+  "importFrom",
+  "importClassesFrom",
+  "importMethodsFrom",
+  "useDynLib"
+)
+
+update_namespace_imports <- function(base_path) {
+  NAMESPACE <- file.path(base_path, "NAMESPACE")
+  if (!made_by_roxygen(NAMESPACE) || !file.exists(NAMESPACE)) {
```

Comment:
doesn't `made_by_roxygen()` run `file.exists()`? Either way it seems odd to run `file.exists()` second here

Reply:
Isn't the logic opposite? We treat a missing `NAMESPACE` as made by roxygen2 so we can override it. Or are you just suggesting I flip the order of the conditional?

Reply:
I'm a bit torn... from readability, it makes more sense to me like (1) check file exists (2) check if it's made by roxygen, but in implementation IINM that would mean `file.exists()` is always run twice, whereas this ordering means it may only be run once.

`file.exists()` is of course not very expensive on most file systems so this is not v. important, sorry for the fly-by comment :)

Reply:
No problem; I had to think this through too when I wrote it, which is probably a sign it needs some refactoring, but it doesn't quite hit the threshold to be worth it.